program/src/main.rs

```diff
@@ -97,13 +76,33 @@ fn main() {
     let target_header_hash =
         B256::from_slice(target_light_block.signed_header.header.hash().as_bytes());
 
-    // ABI-Encode Proof Outputs
+    // Construct a bitmap of the validators from the trusted header that signed off on the new header.
+    // This is used to equivocate slashing in the case that validators are malicious. 256 is chosen
+    // as the maximum number of validators as it is unlikely that Celestia has >256 validators.
+    let mut validator_bitmap = [false; 256];
+    #[allow(clippy::needless_range_loop)]
+    for i in 0..trusted_light_block.signed_header.commit.signatures.len() {
+        if let CommitSig::BlockIdFlagCommit {
+            validator_address: _,
+            timestamp: _,
+            signature: _,
+        } = &trusted_light_block.signed_header.commit.signatures[i]
+        {
+            validator_bitmap[i] = true;
+        }
+    }
```

Comment:
If I understand correctly, this is setting the bitmap for the trusted header. This is not enough to mitigate the discussed issue.

The bitmap should reflect the intersection of the validators that signed all the blocks in the range.

So, for example, if we have 6 validators, and we have:
- block 1: signed by 1,2,3,4,5
- block 2: signed by 3,4,5,6
- block 3: signed by 1,2,3,4,5
- block 4: signed by 4,5,6

Then, the bitmap should be: `[false, false, false, true, true, false]`